installer/windows/Product.wxs

```diff
@@ -83,19 +83,26 @@
     <Property Id="STORJ_STORAGEDIR">STORAGEDIR</Property>
     <Property Id="STORJ_STORAGE">1.0</Property>
     <Property Id="STORJ_BANDWIDTH">2.0</Property>
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Launch SNOBoard" />
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOX" Value="1"/>
```

Comment:
Can we use more meaningful name for that? e.g. `WIXUI_EXITDIALOGOPENDASHBOARDCHECKBOX`

Reply:
It's a built-in property in the standard ExitDialog of WiX Toolset :)

Reply:
oh, then looks good ;)